### PR TITLE
Dark vlm

### PR DESCRIPTION
**Pull request by goodboy**
WIP plotting "dark" volume trades overlayed with normal feed provided by the `ib` backend.

The feed for these are the so called *unreportable trades* as vaguely mentioned in their tick types under id `48`:
https://interactivebrokers.github.io/tws-api/tick_types.html

> Last trade details (Including both "Last" and "Unreportable Last" trades).

---
ToDo:
- [x] separate curve for dark volume overlaid with our current $vlm
- ~[ ] possibly a line overlayed with the OHLC bars to some how indicate the *direction* of these ticks~ (deferred to #266)

### CODE REVIEW

File: piker/fsp/_momo.py [hunk 1]
```diff
@@ -169,10 +170,8 @@ def wma(
     return np.convolve(signal, weights, 'valid')
 
 
-# @piker.fsp.emit(
-#     timeframes=['1s', '5s', '15s', '1m', '5m', '1H'],
-# )
-async def _rsi(
+@fsp
```
Comment (guilledk) [inline piker/fsp/_momo.py:173]: Slick API!

**Review by guilledk — APPROVED** [review]

**Comment by goodboy** [comment]
One last thing is we gotta land https://github.com/goodboy/tractor/pull/295 for the new `NamespacePath` type.

Ping @guilledk

File: piker/fsp/_momo.py (hunk 1 above)
Comment (goodboy) [reply piker/fsp/_momo.py:173]: 💋 thanks amigo

**Comment by goodboy** [comment]
One last note on this was whether to *always* show the dark vlm curve even if no feed is available from the data provider. For now it's just getting set to zero and that zero curve is updated on each step; we should probably have some kind of "symbol info" field that determines whether we draw it at all to avoid the unneeded overhead in most cases.

**Pull request merged**